### ISSUE TITLE
[Accessibility] [Password page] [Contrast] inherent contrast in the "Por favor, digite 12 palavras" text 

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#869]
**Describe the bug**
The contrast in the text is 1.3:!. It should be 4,5:1. The purple background and red font can't contrast enough. The color choose should be reviewed to work better to all people, including all kind of blindness.

**To Reproduce**
1. Go to '[Muda's Password page](https://muda.cambiatus.io/login?redirect=%2Fdashboard)'
2. Check with Colour contrast analyser (cca)
3. See error

**Expected behavior**
The contrast must be 4,5:1

**Screenshots**
![image](https://github.com/user-attachments/assets/3670bedc-3d72-4c56-9d70-3683c6cdeade)

**Cambiatus Env you are using**
Put the URL of the page you are trying to use

**Desktop (please complete the following information):**
 - OS:Windows
- Browser Chrome
- Version 129.0.6668.101 (Versão oficial) 64 bits

**Additional context**

**Comment by ceciliabelem** [comment]
For mapping all defects statistics, I just updated the spreadsheet. Also, there is a graphic to help understand some kinds of bugs and how they can impact users.
[Muda's Report Spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
